modernism/static/_js/modernism.js

```diff
@@ -126,6 +126,24 @@ const setModalImage = (direction) => {
 
 }
 
+//Get the button
+let mybutton = document.getElementById("myBtn");
+// When scroll down 100px from the top of the document, show the button
+window.onscroll = function() {scrollFunction()};
```

Comment:
Please move to the bottom to keep the code structure (definitions on the top, call of functions on the bottom of the file).